mm2src/mm2_main/src/lp_swap/maker_swap.rs

```diff
@@ -901,7 +901,7 @@ impl MakerSwap {
         };
         let msg = SwapMsg::MakerPayment(payment_data_msg);
         let abort_send_handle =
-            broadcast_swap_message_every(self.ctx.clone(), swap_topic(&self.uuid), msg, 600., self.p2p_privkey);
+            broadcast_swap_msg_every(self.ctx.clone(), swap_topic(&self.uuid), msg, 600., self.p2p_privkey);
```

Comment:
One more remark
`broadcast_swap_msg_every(self.ctx.clone(), swap_topic(&self.uuid), msg, MAKER_PAYMENT_TIMEOUT_SEC, self.p2p_privkey);`

Reply:
Done

Reply:
What about the other timeout/interval constants in the current file?
Could I ask you please to rename these using the `_SEC` postfix? It looks like all what had to be done has been done )

Reply:
Done